sdk/eventhub/event-hubs/samples/browserSample/package.json

```diff
@@ -29,7 +29,7 @@
   "sideEffects": false,
   "dependencies": {
     "@azure/event-hubs": "^5.0.0",
-    "@azure/identity": "^1.0.2"
+    "@azure/identity": "2.0.0-beta.3"
```

Comment:
Is the Identity GA for 2.x happening soon? I'm wondering why we're updating samples if we don't want users to bake a beta into their programs (ie, if they're using this as an example of what they have to reference).

Reply:
I think that's a good point. @sadasant Why do we want to update samples' identity dependency?

Reply:
Also in some of the packages, the dependency for the samples is mentioned as a dev-dep in the package. So in making that transition,  there might be situation where samples depend on an identity version other than the version that package takes dev dep on. Thoughts? @sadasant

Reply:
We want to make sure that as many things that use Identity get to use v2 before we release it. v2 will release on September

Reply:
anything that runs on CI will potentially help us catch bugs before we release 🙏

Reply:
For what it's worth, the browser sample is not run on CI. I also am hesitant to depend on a beta in a sample.

Reply:
Is it ok to use beta for node samples, since those run in ci? @chradek

Reply:
Our tests should be using the v2-beta of Identity to help with getting test coverage. Samples is for the end customers to use and should not use beta versions. Also, if we are getting some test coverage in samples that we are not getting in our tests, then we should take another look at our tests.

Reply:
I think the only samples that should use the beta would be samples for require said beta? If this sample works against the GA version of identity, I would think it should target the GA version of identity.

Reply:
> Is it ok to use beta for node samples, since those run in ci?

I'll echo @ramya-rao-a and @xirzec that I don't think samples should use betas unless they need functionality that's only in the beta.

These show up in our published documentation for GA packages, so having a user start by using one of our samples that depends on a beta can lead to a poor experience. We haven't been shy about making breaking changes in between beta versions so I'd hate for a customer to end up depending on something still going through churn because they downloaded an official sample.